src/IndexManager.php

```diff
@@ -21,7 +21,7 @@ class IndexManager implements IndexManagerInterface
     private $indexIfMapping;
     private $normalizer;
 
-    public function __construct(NormalizerInterface $normalizer, EngineInterface $engine, array $configuration)
+    public function __construct($normalizer, EngineInterface $engine, array $configuration)
```

Comment:
Can't you inject the serializer directly?

Reply:
Technically, it is the entire serializer (with both Symfony and jms) but I named it `normalizer` because we use only the part to turn the class into an array, not the part from array to format (json, csv,...)

Do you have something in mind with the serializer?